onoff.d.ts

```diff
@@ -0,0 +1,44 @@
+
+export class Gpio {
+    static HIGH: number;
```

Comment:
Would it be better `HIGH` was a `BinaryValue` rather than a `number`?

Reply:
No, because High is always supposed to be `1`, but so it shouldn't be a number either.

I've added two types `High` and `Low` with the values `1` and `0` respectively. I've also added those along the other types which contain absolute values (high, in, failling ...) at the top of the file so it's easier to maintain in case they need changing.

Reply:
That's a good solution.